### ISSUE TITLE
GetSepaAccounts with DKB / pushTan

### ISSUE THREAD

**Issue by valoricDe** [nemiah/phpFinTS#416]
Hello there.
I am coming from the javascript side (https://github.com/Prior99/fints)
But DKB doesn't like the tan response message.

So I thought I try this library. I used the tanModesAndMedia example. Works like a charm.

So I tried the accounts.php script from docker environment from the shell.
It sends a tan to my tan to go app. But when I login into the app it shows me the error code 9907.
I haven't found this number in the specs. Any idea what could be wrong?

**Comment by Philipp91** [comment]
Does the same TAN2Go app work in other contexts? IIRC, DKB uses that app only very rarely nowadays, because typically they just have you confirm your actions in the main DKB app. So it's conceivable that you haven't used the TAN2Go app at all in a while and it broke due to this.

Are the other TanModes or media that you could select through the phpFinTs APIs?

I assume the error 9907 is displayed by the app and not returned through the FinTS API, as I don't see it in your log/gist. That would explain why it's not in the spec -- it's a DKB-specific error code and you would have to ask their support what's going on. Perhaps you could plug in your phone (assuming it's Android) and run Logcat to see a more detailed error message from the DKB app that could give some clues as to what's going on.

**Comment by valoricDe** [comment]
Yes DKB only uses the Tan2Go app as a fallback, so often you only need it for transfers. But you can login into the account with Tan2Go and I just tested, it works. It is the only method given by the tanModes sample: `[921] => TAN2go`.

I am already in contact with the support. But let's see if they are technical. Do you know if there is already some special DKB handling in this library implemented?

**Comment by Philipp91** [comment]
I don't recall special DKB handling. FWIW, I've been using this library with DKB for years and it works, including the quarterly TAN re-entry.
